README.md

```diff
@@ -83,7 +84,8 @@ Sample message
       "timestamp":"2019-12-29T10:31:14.373202Z",
       "trackId": 42,
       "numberPlate": "IXXX42",
-      "numberPlateOrigin": "AT"
+      "numberPlateOrigin": "AT",
+      "numberPlateRegion":"W"
```

Comment:
I would call it `numberPlateArea` since it is not always a region.
https://commons.wikimedia.org/wiki/File:German_car_registration_plate_labels-DE.svg

Reply:
Makes sense. changed.